en_ID/pages/faq.md

```diff
@@ -11,6 +11,53 @@
 ### What are the eligibility requirements for participation?
 * Generally, there are no strict requirements. Anyone who early in their software engineering career can join this program as long as can commit a significant amount of time to this internship as we mention in the [Duration and time commitment](internship.md#Duration_and_time_commitment). This internship program is intensive, so a time commitment is very important as this program is running virtually.
 
+### How to Make a PR (Pull Request) ?
+
+The very first thing to do before making a PR is creating a new branch for your PR.
+
+The important point for vreating branch is CREATE BRANCH FROM YOUR MASTER BRANCH.
+
+- Open the Terminal
+- Go to your project directory
+- Check current status.
+```
+git status
+```
+![Branch Image](images/Branch-image.png)
+
+Be sure to be on the master Branch of your project.
+- Then create a new breanch from master branch.
+```
+git checkout -b [your branch dir/your branch name]
```

Comment:
somehow this is not aligned with other identation

Reply:
I think it seems okay now. Can you check it?

Reply:
Yes, it is ok now.